cedar-policy-core/src/est.rs

```diff
@@ -342,6 +342,8 @@ impl std::fmt::Display for Clause {
 
 // PANIC SAFETY: Unit Test Code
 #[allow(clippy::panic)]
+// PANIC SAFETY: Unit Test Code
+#[allow(clippy::indexing_slicing)]
```

Comment:
FWIW, I never get see clippy complaining about panics etc in tests. Is this some configuration you've set locally?